sql/telemetry_derived/scalar_percentiles_v1/query.sql

```diff
@@ -0,0 +1,346 @@
+CREATE TEMP FUNCTION udf_zip_arrays(keys ARRAY<FLOAT64>, values ARRAY<FLOAT64>)
+RETURNS ARRAY<STRUCT<key STRING, value FLOAT64>> AS (
+  -- Zips two arrays into a MAP type.
+  (
+    SELECT ARRAY_AGG(record)
+    FROM (
+      SELECT
+        STRUCT<key STRING, value FLOAT64>(CAST(k AS STRING), values[OFFSET(off)]) as record
+      FROM
+        UNNEST(keys) AS k WITH OFFSET off
+    )
+  )
+);
+
+WITH flat_clients_scalar_aggregates AS (
+  SELECT * EXCEPT(scalar_aggregates)
+  FROM clients_scalar_aggregates_v1
```

Comment:
After some discussion with @jklukas, namely https://github.com/mozilla/bigquery-etl/pull/533#discussion_r350391421, it seemed like a better idea to use a date as a dummy partition. So I've filed this issue here: https://github.com/mozilla/bigquery-etl/issues/535

So for the time being, I am not doing partition replacement in the `clients_scalar_aggregates` query, only the most recent data is kept. So the channel/version filter wouldn't be necessary here. However, I think we could make the date partition in `clients_scalar_aggregates` use `submission_date` so that the filter here would be for the most recent submission.

Let me know what you think

Reply:
Perfect, just wanted to make sure. Adding the `submission_date` filter seems fine.